docs/development/releasing.rst

```diff
@@ -197,8 +245,14 @@ packages that use the full bugfix/maintenance branch approach.)
 
       $ python setup.py build sdist
       $ gpg --detach-sign -a dist/astropy-<version>.tar.gz
+      $ twine check dist/*
       $ twine upload dist/astropy-<version>*
 
+.. _post-release-procedure:
```

Comment:
```suggestion

Congratulations!  You have completed the release! Now there are just a few
clean-up tasks to finalize the process.

.. _post-release-procedure:
```

Reply:
Is this sarcasm? 😏

Reply:
Wait, am I supposed to commit that in or not, because I did. :scream:

Reply:
It wasn't intended to be sarcasm, I think, but usually the most tedious, less fun part is the cleanups, updating the contributors page and mailmaps, etc. 🙈